test/end-to-end.js

```diff
@@ -970,6 +972,36 @@ describe('end-to-end', function() {
       });
     });
 
+    it('passes scaffolded tests in development mode', function(done) {
```

Comment:
We are not interested in scaffolded tests, but want to see what an error response looks like. To do that, you need to request a URL that returns an error response.